ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -1,15 +1,26 @@
 #Requires -RunAsAdministrator
 function Get-sbRDSession {
     [OutputType('Custom.SB.RDSession')]
-    [cmdletbinding()]
+    [cmdletbinding(DefaultParameterSetName = 'State')]
```

Comment:
Although I like the use of paramsets, could you ever want to use _all_ params together at all?

Reply:
I gave this a lot of thought and couldn’t think of a scenario where I’d want to find all “Active” sessions then narrow it down to a single user. I figured the user I want is either there or not, regardless of state. 

Suggestions totally welcome though...

Reply:
Fair point